src/PdfReader/index.tsx

```diff
@@ -1,24 +1,82 @@
+import { Document, Page } from 'react-pdf/dist/esm/entry.parcel';
+
 import React from 'react';
 import {
   ColorMode,
   ReaderArguments,
   ReaderReturn,
   ReaderState,
+  WebpubManifest,
 } from '../types';
-import PDFContent from './PdfContent';
 
 type PdfState = ReaderState & {
   type: 'PDF';
-  // some pdf-specific state here, like the instance of the
-  // package you are using if necessary
+  // The PDF information has been fetched
+  fetchSuccess: boolean;
+  // Is the PDF ready for display
+  displayReady: boolean;
```

Comment:
Let me know how else I might make these two things clearer.  The main reason to do this is because there's no way to know how many pages a PDF has until it's loaded.  However, if you're going from the first page of Chapter 3 to to the last page of Chapter 2, we don't want it to flash page 1, so I've held off showing anything until we know that you're loading the correct page.